crates/sui-framework/sources/coin.move

```diff
@@ -179,22 +184,24 @@ module sui::coin {
 
     /// Burn a Coin and reduce the total_supply. Invokes `burn()`.
     public entry fun burn_<T>(c: &mut TreasuryCap<T>, coin: Coin<T>) {
-        burn(c, coin)
+        burn(c, coin);
```

Comment:
Interesting... bit annoying that you have to have this separate entry function just to deal with the `u64` in `burn`

Reply:
Oh! I see now.